### ISSUE TITLE
XXある POS. adj-f? adj-pn? exp?

### ISSUE THREAD

**Issue by JMdictProject** [JMdictProject/JMdictIssues#50]
In the (now deleted) entry 2850815 (名誉ある), Robin raised the issue of the POS for these sorts of terms:
"由緒ある, 価値ある, 良識ある and 勇気ある are [adj-f] only. 伝統ある and 幸ある are [exp,adj-f]. 心ある (which is in the kokugos as 連体) is adj-pn. Should we drop exp? Or make them all adj-pn?"

I guess the same issue arises with entries like 気品のある, which is tagged (exp,adj-f), but I doubt that's really regarded as 連体.

Anyway, I have no problem calling 由緒ある "adj-pn".

**Comment by Marcusjmdict** [comment]
I think [exp,adj] looks good but I've never been clear on the difference between adj-f and adj-pn so I don't have anything to contribute on that.

**Comment by robinjmdict** [comment]
adj-pn corresponds to [連体詞](https://ja.wikipedia.org/wiki/%E9%80%A3%E4%BD%93%E8%A9%9E) – uninflected words that are always prenominal. Only native Japanese words are classified as 連体詞.

We use adj-f for all other prenominal stuff, including gairaigo, kango, expressions and (most) inflected verbs. I don't know whether 由緒ある, 価値ある, 良識ある, etc. are considered 連体詞 or just expressions that are used prenominally. I suspect it's the latter; they're simple sum-of-parts and not in any of the kokugos.

We might want to reserve adj-pn for "true" 連体詞 and use [exp,adj-f] for anything that looks like an expression.

**Comment by ReneMalenfant** [comment]
I agree that 由緒ある should be [exp,adj-f] and 連体詞 should be reserved for 連体詞.
